blocks/library/image/index.js

```diff
@@ -79,18 +79,11 @@ registerBlock( 'core/image', {
 
 		if ( ! url ) {
 			return (
-				<div className="blocks-image is-placeholder">
-					<div className="placeholder__label">
-						<Dashicon icon="format-image" />
-						{ wp.i18n.__( 'Image' ) }
-					</div>
-					<div className="placeholder__instructions">
-						{ wp.i18n.__( 'Drag image here or insert from media library' ) }
-					</div>
+				<Placeholder instructions={ wp.i18n.__( 'Drag image here or insert from media library' ) } icon="format-image" label={ wp.i18n.__( 'Image' ) } className="blocks-image">
```

Comment:
Maybe we could rewrite this like the following to avoid the long line?

```js
<Placeholder 
  instructions={ wp.i18n.__( 'Drag image here or insert from media library' ) } 
  icon="format-image"
  label={ wp.i18n.__( 'Image' ) } className="blocks-image"
>
```